src/Lykke.Service.EthereumSamurai.MongoDb/Repositories/TransactionRepository.cs

```diff
@@ -130,7 +127,7 @@ public async Task<IEnumerable<TransactionModel>> GetAsync(TransactionQuery trans
             MongoDB.Driver.
             IFindFluent<TransactionEntity, TransactionEntity> search = _collection.Find(filter);
             result = new List<TransactionModel>();
-            search = search.Sort(sort);
+            //search = search.Sort(sort);
```

Comment:
Are we sure, that this code is not used in production code and this change not broke something?